packages/generator/src/generators/form-generator.ts

```diff
@@ -42,6 +42,13 @@ export class FormGenerator extends Generator<FormGeneratorOptions> {
       modelNames: this.options.modelNames,
       ModelName: this.options.ModelName,
       ModelNames: this.options.ModelNames,
+      subTemplateValues: this.options.extraArgs?.map((arg: string) => {
+        const [valueName] = arg.split(":")
+        return {
+          fieldName: singleCamel(valueName),
+          FieldName: singlePascal(valueName).replace(/(?!^)([A-Z])/g, " $1"),
```

Comment:
Also add the following which will put spaces between the words.
- `field_name`
- `Field_name`
- `Field_Name` - ex: `myFirstName` -> `My First Name`

Reply:
Still todo